llarp/crypto/crypto.hpp

```diff
@@ -132,7 +132,8 @@ namespace llarp
         return m_crypto;
       }
 
-      throw std::logic_error("Cryptomanager::instance() was undefined");
+      assert(false && "Cryptomanager::instance() was undefined");
+      abort();
```

Comment:
probably redundant here

Reply:
since `assert(3)` on Windows branches to `abort(3)`, and pops up a dialogue with the assert message. Simply calling `abort` triggers a generic error message that isn't terribly useful

Reply:
Only if asserts are enabled.